apis/v1alpha1/httproute_types.go

```diff
@@ -432,6 +432,24 @@ const (
 // HTTPRequestHeaderFilter defines configuration for the RequestHeaderModifier
 // filter.
 type HTTPRequestHeaderFilter struct {
+	// Set overwrites the request with the given header (name, value)
+	// before the action.
+	//
+	// Input:
+	//   GET /foo HTTP/1.1
+	//   My-Header: ABC
```

Comment:
Can we lower case this?
Header field names are case-insensitive but keeping this example consistent throughout the documentation would be ideal.